examples/01_plotting/plot_demo_glass_brain.py

```diff
@@ -20,10 +20,15 @@
 # demo glass brain plotting
 from nilearn import plotting
 
+# Whole brain saggital cuts
```

Comment:
To make the HTML view of the example simpler to comprehend, I think that it would be good to have 3 different "cells" in this example, one per plot.